### PR TITLE
Add homework 1 & 2

### PR DESCRIPTION
**Pull request by Stallind**
Homework #1 

Read name, surename, age, weight (in kg) and height (in cm) from console.
1) Print all the info based on the example message below:
Tom Jefferson is 19 years old, his weight is 50 kg and his height is 156.5 cm. 

2) Calculate and print body-mass index (BMI)
3) Do 1 and 2 for another person.

### CODE REVIEW

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Program.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4832091&bid=16052660)

**Comment by Alumniminium** [comment]
Next time, please give a short description of what your pull request contains.

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -6,7 +6,47 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            int entry = 1;
+            const int maxEntries = 3;
+
+            while (entry < maxEntries)
```
Comment (Alumniminium) [inline Src/BootCamp.Chapter/Program.cs]: This would be the perfect opportunity for a `for` loop :)

`for(int i = 0; i<maxEntries; i++)`

**Comment by Stallind** [comment]
> Next time, please give a short description of what your pull request contains/what your code does.

Fixed =)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Program.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4832091&bid=16052660)

**Comment by Alumniminium** [comment]
Assigning @Almantask for final merge :)

**Pull request closed without merging**